xml/System/Object.xml

```diff
@@ -762,15 +764,15 @@ and the <xref:System.IDisposable> interface. The <xref:System.IDisposable.Dispos
   
  For two objects `x` and `y` that have identical runtime types, `Object.ReferenceEquals(x.GetType(),y.GetType())` returns `true`. The following example uses the <xref:System.Object.GetType%2A> method with the <xref:System.Object.ReferenceEquals%2A> method to determine whether one numeric value is the same type as two other numeric values.  
   
```

Comment:
this page is also missing the interactive note

Reply:
Added.